Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:4-onbuild
```

Comment:
I tried, but It didn't build... it was more to test that drone was working fine than to have the proper dockerfile